genesis/src/main.rs

```diff
@@ -25,22 +28,44 @@ use std::time::{Duration, Instant};
 
 pub const BOOTSTRAP_LEADER_LAMPORTS: u64 = 42;
 
+pub const VOTE_PROGRAM_NAME: &str = "vote_program";
+pub const STAKE_PROGRAM_NAME: &str = "stake_program";
+pub const SYSTEM_PROGRAM_NAME: &str = "system_program";
+
 pub enum AccountFileFormat {
     Pubkey,
     Keypair,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PrimordialAccountDetails {
+    balance: u64,
+    owner: String,
+}
+
+impl PrimordialAccountDetails {
+    pub fn get_program_id(self) -> Result<Pubkey, String> {
```

Comment:
Let's remove this mapping as it's something that we'll now need to maintain as new owners are added. Instead the full base58 owner pubkey can be provided in the file.

Reply:
@mvines what if it is invalid?

Reply:
That's ok.  We don't need to enforce that here IMO.

If it's invalid then you get a thousand lashings for creating a bad genesis block in real life already!   Genesis block construction is something that ought to be heavily scrutinized and validated by humans before the cluster is booted (if the cluster is important, like a mainnet)

Reply:
done